docs/Home.md

```diff
@@ -11,8 +11,8 @@ OAuth for Spring Security provides an [OAuth](http://oauth.net)
 implementation for
 [Spring Security](http://projects.spring.io/spring-security/).
 Support is provided for the implementation of OAuth providers and
-OAuth consumers. There is support for [Oauth 1(a)](oauth1.html) (including
-[two-legged OAuth](twolegged.html), a.k.a. "Signed Fetch") and for
+OAuth consumers. There is support for [Oauth 1(a)](oauth1.md) (including
```

Comment:
@annaarun123 In case you are not aware, the `md` files get converted to `html` by [GitHub pages](https://help.github.com/en/github/working-with-github-pages/about-github-pages). The existing `html` links are all valid. You can confirm it from the [main page](https://projects.spring.io/spring-security-oauth/docs/Home.html).